Makefile

```diff
@@ -114,9 +114,10 @@ e2e-bootstrap: install-helm
 	helm install csi-driver-nfs ./charts/latest/csi-driver-nfs --namespace kube-system --wait --timeout=15m -v=5 --debug \
 	--set image.nfs.repository=$(REGISTRY)/$(IMAGENAME) \
 	--set image.nfs.tag=$(IMAGE_VERSION) \
-	--set image.nfs.pullPolicy=Always
-	--set controller.logLevel=8
-	--set node.logLevel=8
+	--set image.nfs.pullPolicy=Always \
+	--set controller.logLevel=8 \
+	--set node.logLevel=8 \
+	--disable-openapi-validation
```

Comment:
it's not necessary to make it as default, only enable this if `disable-openapi-validation` flag is defined.